server/routes/status-update/statusUpdateController.ts

```
@@ -21,6 +22,11 @@ export default class StatusUpdateController {
       const prisonerData = await this.prisonerDetailsService.loadPrisonerDetailsFromParam(req, res, true)
       const selectedPathway = req.query.selectedPathway as string
       const validationError = req.flash('validationError')?.[0]
+      const supportNeedsEnabled = await getFeatureFlagBoolean(FEATURE_FLAGS.SUPPORT_NEEDS)
+
+      if (supportNeedsEnabled) {
```

Can you also throw an error is someone calls the postStatusUpdate endpoint when the flag is on?